### DISCUSSION TITLE
[Feature] recluster all unassigned / unnamed faces

### DISCUSSION THREAD

**Discussion by SomeThink1729** [immich-app/immich#13926] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

After manually merging a part of unassigned people to the right people, the cluster of the person improved and it should be able to better recluster the still missing unnamed faces. 
I'd like to have the feature of all non named people in the admin panel at jobs, like refresh for face detection, recluster all non named ... for Face Recognition.
Resetting all is no option as it destroys all information / progress with face detection and metadata.
 Increasing the distance is no option as well, as it puts siblings together.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#5507

**Reply by SomeThink1729** [reply]
You're right, I only searched for cluster and couldn't find any that matched
